src/app/course/course-practice/course-practice.component.html

```diff
@@ -0,0 +1 @@
+<app-concept-map [currCourse]="course"></app-concept-map>
```

Comment:
What is this component?

Reply:
This component just displays the course map. Eventually, it will have to switch between the list view and the concept map view, so I thought it was fine if it has its own component.